lib/rucio/web/rest/webpy/v1/did.py

```diff
@@ -1099,6 +1099,11 @@ def DELETE(self, scope, name):
    Web service startup
 ----------------------"""
 
+logfile = open('/tmp/rucio_log_jtp', 'w')
+for url in URLS:
+    logfile.write(url + "\n")
+logfile.close()
```

Comment:
should this still be in here? debug code?

Reply:
Well spotted, it should not be there. I will remove it.